src/hotspot/share/runtime/mutexLocker.cpp

```diff
@@ -409,3 +422,36 @@ void print_owned_locks_on_error(outputStream* st) {
   }
   if (none) st->print_cr("None");
 }
+
+void print_lock_ranks(outputStream* st) {
+  st->print_cr("VM Mutex/Monitor ranks: ");
```

Comment:
Do we want to clarify this is not the complete set of Mutexes in the VM?

Reply:
hs_err does not clarify this in the similar printout ("VM Mutex/Monitor currently owned by a thread: " in the method above), so it seems to be implied these are only the known mutexes. So, I left it as is.

Reply:
it's a lot harder to get the full set of mutexes, which is unfortunate, but this is good.